files/debugger.cpp

```diff
@@ -1,7 +1,9 @@
 #include <iostream>
 
 int main() {
-    int a, b, c;
+    int a;
+    int b;
+    int c;
```

Comment:
```suggestion
    int b{};
    int c{};
```
Hab jetzt auch rausgefunden wie ich aufm Handy suggestions für mehrere Lines machen kann